MkDocs/contribute/docs/prereqs.md

```diff
@@ -54,15 +88,15 @@ Dectivate Virtual Environment
 
 Clone the github repository down to your local machine:
 
-    cd ~
-    mkdir Github
-    cd ~/Github
-    git clone git@github.com:IBM/monitor-hands-on-lab.git
-    cd monitor-hands-on-lab
+    $ cd ~
```

Comment:
Leave as is we don't want $ in scripts because it makes it harder to copy and paste.

Reply:
I'll remove the "$ " in all the code blocks.